docs/source/docs/registered_classes.md

```diff
@@ -0,0 +1,47 @@
+# Registered Classes
+
+The following tables contain all registered classes with their categories and keys.
+
+## Backbone
+
+The category is `backbone_class`. Registered keys and values are as following.
+
+| key      | value                                         |
+| :------- | :-------------------------------------------- |
+| "global" | `deepreg.model.backbone.global_net.GlobalNet` |
+| "local"  | `deepreg.model.backbone.local_net.LocalNet`   |
+| "unet"   | `deepreg.model.backbone.u_net.UNet`           |
+
+## Model
```

Comment:
hide the "model" category now, it would be easier to add later - not modifying.

Reply:
not here, this file is generated automatically.